src/main/webapp/resources/js/utilities/restriction-utilities.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @fileoverview utilities to help with metadata restrictions
+ */
+
+/**
+ * Default colours for restriction level
+ */
+const COLOURS = {
+  LEVEL_1: "green",
+  LEVEL_2: "orange",
+  LEVEL_3: "volcano",
+  LEVEL_4: "red",
+};
+
+const LEVELS = ["LEVEL_1", "LEVEL_2", "LEVEL_3", "LEVEL_4"];
+
+/**
+ * Get a colour assoc with a specific restriction. Use for consistency.
+ * @param restriction
+ * @returns {*}
+ */
+export const getColourForRestriction = (restriction) => COLOURS[restriction];
+
+/**
+ * Compare two metadata restrictions.
+ *
+ * Right now levels can be easily compared by their string values (e.g. "LEVEL_1 < LEVEL_2"
+ * would evalueat to true).  This helper function is here incase those values ever change
```

Comment:
Typo in `evaluate`

Reply:
Updated in 2d474e5bd01b37713bb8aaa910615dc0316d0217